R/case.R

```diff
@@ -1,21 +1,41 @@
-#' Convert string to upper case, lower case, title case, or sentence case
+#' Convert string to upper case, lower case, title case, sentence case, pascal
+#' case, camel case, kebab case, or snake case
 #'
 #' * `str_to_upper()` converts to upper case.
 #' * `str_to_lower()` converts to lower case.
 #' * `str_to_title()` converts to title case, where only the first letter of
 #'   each word is capitalized.
 #' * `str_to_sentence()` convert to sentence case, where only the first letter
 #'   of sentence is capitalized.
+#' * `str_to_pascal()` converts to pascal case, where only the first letter of
```

Comment:
I'm not sure Pascal case is particularly well known (at least I didn't recognise it) so how about it make it an argument to `str_camel_case()`, e.g. `first_upper = TRUE`?